proto/cosmos/distribution/v1beta1/tx.proto

```diff
@@ -64,7 +66,8 @@ message MsgSetWithdrawAddressResponse {}
 // MsgWithdrawDelegatorReward represents delegation withdrawal to a delegator
 // from a single validator.
 message MsgWithdrawDelegatorReward {
-  option (cosmos.msg.v1.signer) = "delegator_address";
+  option (cosmos.msg.v1.signer)            = "delegator_address";
+  option (cosmos.msg.legacy_amino.v1.name) = "cosmos-sdk/MsgWithdrawDelegatorReward";
```

Comment:
Here it also does not match:

```
cdc.RegisterConcrete(&MsgWithdrawDelegatorReward{}, "cosmos-sdk/MsgWithdrawDelegationReward", nil)
```

Reply:
Thanks for catching these 2! I'll try to work on some validation (i.e. make sure proto annotations match amino RegisterConcrete) right after this PR, in https://github.com/cosmos/cosmos-sdk/issues/13405#issuecomment-1284032757